Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:3
+FROM python:3.6.8-stretch
```

Comment:
python:3 is pointing to buster (debian 10), 3.7.4, but I suspect that it used to be stretch + 3.6 when we first made this image. Being more specific in the tag is a good idea here, but it'd also be interesting to work out why this doesn't seem to compile on buster. We should be able to properly support this with essentia

Reply:
Sorry, I was not clear enough.
python:3.6.8-stretch points to Debian Stretch, which uses the old version of ffmpeg libraries (e.g. libavcodec57). 
The extractor is using an old version of Essentia, that is based on the old versions of the ffmpeg related libraries.
In order to keep compatibility and since we use a fixed version of Essentia, using the python:3.6.8-stretch enables to install the same versions as before.